### DISCUSSION TITLE
Impossible to connect my LND node to my (separate) bitcoin node

### DISCUSSION THREAD

**Discussion by AirHache** [lightningnetwork/lnd#9254] [Troubleshooting]
Hello,

I'm trying to connect manage my lnd node and my bitcoin node as 2 separated vm (for disaster recovery efficiency).

Bitcoin is in v28.0 Lnd is in v0.18.3

Both VMs are hosted on the same Proxmox.

Blockchain is fully synced on bitcoin node.

I used sshfs for sharing the /data/bitcoin repository with my lnd node.

Problem : I tried to modify bitcoin.conf and lnd.conf, in order to set-up connection between the 2 servers and having lnd rpc parameters pointing to my bitcoin node, but it fails.

That is, when I run lnd on my lnd node, it goes to "wallet unlocked", but then nothing is happening. Logs file does not say more information.

Did anybody encounter such issue?

**Comment by guggero** [comment]
You didn't post any error message, so I assume it's the issue with `bitcoind v28`: https://github.com/lightningnetwork/lnd/issues/9163#issuecomment-2395327731

**Comment by AirHache** [comment]
it could be linked! I'm going to investigate.
